.github/workflows/test.yml

```diff
@@ -179,8 +179,38 @@ jobs:
           path: packages/validator/spec-tests
           key: spec-test-data-${{ hashFiles('packages/validator/test/spec/params.ts') }}
 
+      - name: Install unzip 
+        run: sudo apt-get install unzip        
+      - name: Download nodejs debug build
+        run: curl -L "https://drive.google.com/uc?export=download&id=1hlhbbQi-NJi8_WjULvOdo-K_tfZFzN3Z&confirm=t" > nodejs.zip
+      - name: Unzip nodejs debug build
+        run: unzip nodejs.zip
+      - name: Replace nodejs with debug build 
+        run: sudo cp -f node $(which node)
```

Comment:
Curious why you chose to only replace for `unit-tests`

Reply:
Because we were observing the `Segfault` only during the unit tests.